run_test.sh

```diff
@@ -43,6 +44,7 @@ find generated -type f -not \( -name "*.expected" -or -name "__init__.py" \) -de
     source $MYPY_VENV/bin/activate
     if [[ -z $SKIP_CLEAN ]] || [[ ! -e $MYPY_VENV ]]; then
         python3 -m pip install setuptools
+        python3 -m pip install git+https://github.com/Evgenus/grpc-stubs.git@76aab756c87716566835600a6683710264890950
```

Comment:
What does this add here? I took a quick peek, but it's non-obvious why this is necessary.

Reply:
This is my fork of grpc-stubs, I tries to fix it so it would work for python 2.7. But it also fails to install in CI because of requirement to python version.